File: app/src/main/java/com/example/vignesh/ktu/MainActivity.java
package com.example.vignesh.ktu;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity implements View.OnClickListener{

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
    }

    @Override
    public void onClick(View view) {
        switch (view.getId()){
            case R.id.button1:
                Intent i1=new Intent(this,semester1.class);
                startActivity(i1);
                break;
            case R.id.button2:
                Intent i2=new Intent(this,semester2.class);
                startActivity(i2);
                break;
            case R.id.button3:
                Intent i3=new Intent(this,semester3.class);
                startActivity(i3);
                break;
            case R.id.button4:
                Intent i4=new Intent(this,semester4.class);
                startActivity(i4);
                break;
            case R.id.button5:
                Intent i5=new Intent(this,semester5.class);
                startActivity(i5);
                break;
            case R.id.button6:
                Intent i6=new Intent(this,semester6.class);
                startActivity(i6);
                break;
            case R.id.button7:
                Intent i7=new Intent(this,semester7.class);
                startActivity(i7);
                break;
            case R.id.button8:
                Intent i8=new Intent(this,semester8.class);
                startActivity(i8);
                break;
            default:
                break;
        }

        }
    }

File: app/src/main/java/com/ktulive/actvities/MainActivity.kt
package com.ktulive.actvities

import android.content.Context
import android.graphics.Color
import android.os.Bundle
import android.support.design.widget.NavigationView
import android.support.v4.view.GravityCompat
import android.support.v7.app.ActionBarDrawerToggle
import android.support.v7.app.AppCompatActivity
import android.view.Menu
import android.view.MenuItem
import com.ktulive.R
import kotlinx.android.synthetic.main.activity_main.*
import kotlinx.android.synthetic.main.app_bar_main.*
import android.graphics.drawable.ColorDrawable
import android.support.v4.app.FragmentTransaction
import android.view.View
import android.widget.ImageView
import android.widget.TextView
import android.widget.Toast
import com.google.firebase.auth.FirebaseAuth

import com.ktulive.GeneralData
import com.squareup.picasso.Picasso
import de.hdodenhof.circleimageview.CircleImageView
import kotlinx.android.synthetic.main.nav_header_main.*
import org.w3c.dom.Text


class MainActivity : AppCompatActivity(), NavigationView.OnNavigationItemSelectedListener {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setSupportActionBar(toolbar)


        var ft:FragmentTransaction  = supportFragmentManager.beginTransaction()
        ft.replace(R.id.fragment_location,HomeFragment())
        ft.commit()
        toolbar!!.background  =(ColorDrawable(Color.parseColor("#ffffff")))
        val toggle = ActionBarDrawerToggle(
                this, drawer_layout, toolbar, R.string.navigation_drawer_open, R.string.navigation_drawer_close)
        drawer_layout.addDrawerListener(toggle)
        toggle.syncState()
        nav_view.setNavigationItemSelectedListener(this)



        var navigationView:NavigationView = findViewById(R.id.nav_view) as NavigationView
        var header:View = navigationView.getHeaderView(0)
       var  display = header.findViewById(R.id.displayName)as TextView
       var  email = header.findViewById(R.id.email)as TextView
       var  imageView = header.findViewById(R.id.imageView)as CircleImageView
        display.setText((FirebaseAuth.getInstance().currentUser?.displayName))
        email.setText((FirebaseAuth.getInstance().currentUser?.email))
       var photoURL =  FirebaseAuth.getInstance().currentUser?.photoUrl.toString()
        Toast.makeText(applicationContext,photoURL,Toast.LENGTH_SHORT).show()
        Picasso.with(applicationContext).load(FirebaseAuth.getInstance().currentUser?.photoUrl.toString()).into(imageView);


    }

    override fun onBackPressed() {
        if (drawer_layout.isDrawerOpen(GravityCompat.START)) {
            drawer_layout.closeDrawer(GravityCompat.START)
        } else {
            super.onBackPressed()
        }
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        // Inflate the menu; this adds items to the action bar if it is present.
        menuInflater.inflate(R.menu.main, menu)
        return true
    }

    override fun onOptionsItemSelected(item: MenuItem): Boolean {
        // Handle action bar item clicks here. The action bar will
        // automatically handle clicks on the Home/Up button, so long
        // as you specify a parent activity in AndroidManifest.xml.
        when (item.itemId) {
            R.id.action_settings -> return true
            else -> return super.onOptionsItemSelected(item)
        }
    }

    override fun onNavigationItemSelected(item: MenuItem): Boolean {
        // Handle navigation view item clicks here.
        when (item.itemId) {
            R.id.contact_us -> {
//                var fragment  =  ContactUsFragment()
                var ft = supportFragmentManager.beginTransaction()
                ft.replace(R.id.fragment_location,ContactUsFragment())
                ft.addToBackStack("1")
                ft.commit()



            }
            R.id.home_se -> {
                var ft = supportFragmentManager.beginTransaction()
                ft.replace(R.id.fragment_location,HomeFragment())
                ft.addToBackStack("1")
                ft.commit()
            }
        }

        drawer_layout.closeDrawer(GravityCompat.START)
        return true
    }
}
